cmake/ConfigureCompiler.cmake

```
@@ -199,6 +199,8 @@ else()
   endif()
   if (CLANG)
     add_compile_options()
+    # Clang has link errors unless `atomic` is specifically requested.
+    add_link_options(-latomic)
```

```suggestion
    if(NOT APPLE)
    add_link_options(-latomic)
    endif()
```

It seems that this breaks on MacOS.